test/cases/alias/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,57 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
It's a shame that Github can't recognise this is just a subset of the previous snapshot file, but I think it's OK.